onnx/backend/test/case/model/sequence.py

```diff
@@ -330,14 +330,15 @@ def make_graph(
         expect(model, inputs=[x], outputs=[out], name="test_sequence_model7")
 
         #8th testcase - split zero length
-        seq_split_node = onnx.helper.make_node('SplitToSequence', ['X'], ['seq_1'])
+        seq_split_node = onnx.helper.make_node('SplitToSequence', ['X', 'Splits'], ['seq_1'])
         seq_len_node = onnx.helper.make_node('SequenceLength', ['seq_1'], ['len'])
 
-        tensor_shape = []  # type: ignore
-        len_shape = []  # type: ignore
+        tensor_shape = ['n']  # type: ignore
+        splits_shape = [3]  # type: ignore
 
         x = np.array([]).astype(np.float32)
-        out_len = np.int64(0)
+        splits = np.array([0, 0, 0]).astype(np.int64)
```

Comment:
Is this a valid `splits`? From the [spec](https://github.com/onnx/onnx/blob/master/docs/Operators.md#splittosequence), I see
```
the sum of entries in 'split' must be equal to the dimension size of input tensor on 'axis'.
```